src/typescript.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+var _ = require('underscore-plus')
+var crypto = require('crypto')
+var path = require('path')
+
+var defaultOptions = {
+  target: 1,
+  module: 'commonjs',
+  sourceMap: true
+}
+
+var TypeScriptSimple = null
+var typescriptVersionDir = null
+
+exports.shouldCompile = function() {
+  return true
+}
+
+exports.getCachePath = function(sourceCode) {
+  if (typescriptVersionDir == null) {
+    var version = require('typescript-simple/package.json').version
+    typescriptVersionDir = path.join('ts', createVersionAndOptionsDigest(version, defaultOptions))
+  }
+
+  return path.join(
+    typescriptVersionDir,
+    crypto
+      .createHash('sha1')
+      .update(sourceCode, 'utf8')
+      .digest('hex') + ".js"
```

Comment:
Might be nice to use consistent quotes on this line, looks like single is the pattern in this file